prestart.sh

```diff
@@ -1,17 +1,15 @@
 #!/bin/sh
 set -e
 
+# This script is run before the server application starts by the tiangolo/uvicorn-gunicorn
+# image. See: https://github.com/tiangolo/uvicorn-gunicorn-docker#custom-appprestartsh
+
 export PGUSER PGPASSWORD PGDATABASE PGHOST POSTGRES_URI
 
+# Wait for the database to be ready
 while ! PGDATABASE=postgres psql --quiet -c "select * from user" ; do
     sleep 1;
 done
 
-echo 'Creating and migrating database'
-# in spin the database already exists
-PGDATABASE=postgres psql -c "create database nmdc_a;" || true
-
-
-PGDATABASE=postgres psql -c "create database nmdc_b;" || true
```

Comment:
Does removing this add an extra step if setting up a new environment in spin? I know that doesn't happen often but I don't see a real reason to remove these lines if they might still have utility outside of the development `docker-compose` workflow.

Reply:
I waffled on removing those lines. On one hand I don't think it's really the _right_ place to do database initialization, but on the other hand I guess it's maybe a useful failsafe measure. And they don't really hurt anything other than adding a bit of noise to the application logs at startup. Okay, I guess I've convinced myself to put them back in.